src/main/resources/io/jenkins/docker/connector/DockerComputerJNLPConnector/config.jelly

```diff
@@ -10,5 +10,14 @@
         <f:textbox/>
     </f:entry>
 
+    <f:entry title="${%Jenkins URL}" field="jenkinsUrl">
```

Comment:
I don't get the goal of this parameter. Jenkins external URL can be configured globaly, and tunnel option is designed to cover reverse-proxy/firewall constraints

Reply:
Hi @ndeloof,
I've added that parameter (actually just exposed it, since it was there before for the sake of Unit Tests), just because it existed in both kubernetes-plugin and aws-ecs-plugin, so I thought it would be nice as standard way to override the current Jenkins URL. 
It is not related to this pull request, so I can remove it if you think it is not needed. 
As a side comment, sometime you don't have a real DNS name, but just an IP and you don't want it to be defined in the global JENKINS configuration, since it is affecting other plugins. IMHO, have this parameter as an optional one would be nice, but again, I can remove it if think it should be removed.